### DISCUSSION TITLE
Whatsapp bro!

### DISCUSSION THREAD

**Discussion by pujayana** [pujayana/getting-learner#7] [Q&A] [answered]
Thanks for review this repository!

**Comment by pjyn-dzgn** [comment, marked as answer]
Yoo! you're welcome bro!

**Reply by pujayana** [reply]
Very nice reponses!

**Reply by pujayana** [reply]
Nice

**Comment by pjyn-dzgn** [comment]
> Thanks for review this repository!

Let's do together~

**Reply by pujayana** [reply]
Yoo, how about learn vue & golang?

**Reply by pjyn-dzgn** [reply]
Wow that's a good idea sir

**Comment by pujayana** [comment]
Thank you brother!
